packages/common-all/src/vault.ts

```diff
@@ -147,9 +147,10 @@ export class VaultUtils {
     const normPath = this.normPathByWsRoot({
       wsRoot,
       fsPath,
-    });
+    }).trim();
+    const unixPath = normPath.split(path.sep).join(path.posix.sep);
```

Comment:
can we extract this into a utility method since we use this in a number of places? (eg. grep `normalize` in the codebase). would put in `[[../packages/common-server/src/files.ts]]` as `normalizePath`

Reply:
found an existing utility for this in common-all `normalizeUnixPath`